# Admittance parameters for mass damper systems 

mass_matrix: [6,0,0,0,0,0,
              0,6,0,0,0,0,
              0,0,6,0,0,0,
              0,0,0,2,0,0,
              0,0,0,0,2,0,
              0,0,0,0,0,2]

damping_matrix: [40,0,0,0,0,0,
                 0,40,0,0,0,0,
                 0,0,40,0,0,0,
                 0,0,0,15,0,0,
                 0,0,0,0,15,0,
                 0,0,0,0,0,15]

# this will limit the workspace ([x_min x_max y_min y_max z_min z_max])
workspace_limits: [0.45, 1.5, -1.3, 1.3, 0.10, 1.4]

# joint limit equal to 2*pi
joint_limits: [6.283, 6.283, 6.283, 6.283, 6.283, 6.283]

maximum_velocity: [1.57, 1.57, 1.57, 1.57, 1.57, 1.57]
maximum_acceleration: [2.61, 2.61, 2.61, 2.61, 2.61, 2.61]
